AFQ/api/group.py

```diff
@@ -187,7 +187,9 @@ def __init__(self,
             "Name": bids_description["Name"],
             "BIDSVersion": bids_description["BIDSVersion"],
             "PipelineDescription": {"Name": "pyAFQ",
-                                    "Version": pyafq_version}}
+                                    "Version": pyafq_version},
+            "GeneratedBy": [{"Name": op.basename(self.afq_path),
+                             "Version": pyafq_version}]}
```

Comment:
Can you really have both of these side-by-side?

Reply:
I left the PipelineDescription for compatibility. I think it's just going to be extra and should be deleted in the near future.